m0t1fs/linux_kernel/ut/file.c

```diff
@@ -996,7 +996,7 @@ static void target_ioreq_test(void)
 
 	/* Addition of parity buffer */
 	buf = map->pi_paritybufs[page_id(0)]
-		[LAY_N % data_col_nr(pdlay)];
+		[LAY_N % layout_n(pdlay)];
```

Comment:
This can fit in a single line. It will not exceed 80 chars.

Reply:
Yes, fixed.